client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.css

```diff
@@ -0,0 +1,3 @@
+.panel {
+  /* Main editing window styling */
+}
```

Comment:
For now, can you comment out this entire thing?

Reply:
Sure!